compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -175,10 +175,14 @@ int entry(int argc, char **argv)
           "destination_tensor_name(string)");
 
   arser.add_argument("--input_type")
-    .help("Input type of quantized model (uint8, int16, or float32)");
+    .help("Input type of quantized model (uint8, int16, int32, int64, float32, or bool). For "
```

Comment:
Previously, we only accepted u8, s16, and fp32, and handle other types implicitly (e.g., int32, int64, bool input is ignored).

To handle individual setting of dtypes of multiple inputs, the exact dtype of each input has to be given.